sphinx-doc/features.rst

```diff
@@ -0,0 +1,139 @@
+
+Python package
+--------------
+
+HOOMD-blue is a Python package and is designed to interoperate with other packages in the scientific
+Python ecosystem and to be extendable in user scripts. To enable interoperability, all operations
+provide access to useful computed quantities as properties in native Python types or numpy arrays
+where appropriate. Additionally, `State <hoomd.State>` and `md.force.Force` provide direct access to
+particle properties and forces using Python array protocols. Users can customize their simulation or
+extend HOOMD-blue with functionality implemented in Python code by subclassing `trigger.Trigger`,
+`variant.Variant`, `hoomd.update.CustomUpdater`, `hoomd.write.CustomWriter`,
+`hoomd.tune.CustomTuner`, or by using the HOOMD-blue API in combination with other Python packages
+to implement methods that couple simulation, analysis, and multiple simulations (such as umbrella
+sampling).
+
+.. seealso::
+
+    Tutorial: :doc:`tutorial/04-Custom-Actions-In-Python/00-index`
+
+CPU and GPU devices
+-------------------
+
+HOOMD-blue can execute simulations on CPUs or GPUs. Typical simulations run more efficiently on
+GPUs for system sizes larger than a few thousand particles, although this strongly depends on the
+details of the simulation. The provided binaries support NVIDIA GPUs. Build from source to enable
+preliminary support for AMD GPUs. CPU support is always enabled. GPU support must be enabled at
+compile time with the ``ENABLE_GPU`` CMake option (see :doc:`building`). Select the device to use at
+run time with the `device <hoomd.device>` module. Unless otherwise stated in the documentation,
+**all** operations and methods support GPU execution. At runtime, `hoomd.version.gpu_enabled` indicates
+whether the build supports run time compilation.
+
+MPI
+---
+
+HOOMD-blue can use the message passing interface (MPI) to execute simulations in less time using
+more than one CPU core or GPU. Unless otherwise stated in the documentation, **all** operations and
+methods support MPI parallel execution. MPI support is optional, requires a compatible MPI library,
+and must be enabled at compile time with the ``ENABLE_MPI`` CMake option (see :doc:`building`).
+At runtime, `hoomd.version.mpi_enabled` indicates whether the build supports MPI.
+
+.. seealso::
+
+    Tutorial: :doc:`tutorial/03-Parallel-Simulations-With-MPI/00-index`
+
+Threading
+---------
+
+Some operations in HOOMD-blue can use multiple CPU threads in a single process. Control this with
+the `device.Device.num_cpu_threads` property. In this release, threading support in HOOMD-blue is
+very limited and only applies to implicit depletants in `hpmc.integrate.HPMCIntegrator`, and
+`hpmc.pair.user.CPPPotentialUnion`. Threading must must be enabled at compile time with the
+``ENABLE_TBB`` CMake option (see :doc:`building`). At runtime, `hoomd.version.tbb_enabled` indicates
+whether the build supports MPI.
```

Comment:
Reading this I'm confused about the discrepancy between the flag saying TBB and the description saying MPI again.

Reply:
Right, copy and paste bug. Fixed.